pyproject.toml

```diff
@@ -5,11 +5,13 @@ description = ""
 authors = ["davidtnfsh <davidtnfsh@gmail.com>"]
 
 [tool.poetry.dependencies]
-python = "^3.7"
+python = "^3.7.1"
```

Comment:
Ahy reason 3.7.0 is being excluded?

Reply:
I barely remember. Let me have a quick trial and keep you posted. My gut feeling shows it's for some package dependency.

Reply:
```
  The current project's Python requirement (3.7.0) is not compatible with some of the required packages Python requirement:                                                                   
    - pandas requires Python >=3.7.1, so it will not be satisfied for Python 3.7.0                                                                                                            
                                                                                                                                                                                              
  Because pandas (1.2.3) requires Python >=3.7.1                                                                                                                                              
   and no versions of pandas match >1.2.3,<2.0.0, pandas is forbidden.
  So, because python-project-template depends on pandas (^1.2.3), version solving failed.
```